README.md

```diff
@@ -1,2 +1,6 @@
 # gae-java
 Support for Spine-based Java apps running under Google App Engine.
+
+GAE Datastore startup script is named `start-datastore.sh`. On windows, make the same `.bat` file or
```

Comment:
Please write concrete steps for configuring [here](https://github.com/SpineEventEngine/gae-java/wiki/Configuring-Local-Datastore-Environment). And remove old steps from wiki.